obj/createTask.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    //DOM elements
+    submitButton : '[type="submit"]',
+    taskName : '[id="taskDetailsForm.name"]',
+    taskDescription : '[id="taskDetailsForm.description"]',
+    taskBill : '.Select-value',
+    taskBillSelect : '[aria-label="?"]',
+    taskBillSelected : '[aria-selected="true"]',
+    taskRate : '[id="taskDetailsForm.rate"]',
+    taskPage : '.main-nav__link--active',
```

Comment:
What is a purpose of this locator? Display active page? Because it is used elsewhere to navigate to task page. Overall this locator could be used as indicator for active page and defined in common object or any other. And for navigation something more concrete should be used